Annex60/Fluid/MixingVolumes/BaseClasses/PartialMixingVolume.mo

```diff
@@ -157,13 +158,76 @@ It is used as the base class for all fluid volumes of the package
 <a href=\"modelica://Annex60.Fluid.MixingVolumes\">
 Annex60.Fluid.MixingVolumes</a>.
 </p>
-
+<p>
+To increase the numerical robustness of the model, the parameter
+<code>prescribedHeatFlowRate</code> can be set by the user.
+This parameter only has an effect if the model has exactly two fluid ports connected,
+and if it is used as a steady-state model.
+Use the following settings:
+</p>
+<ul>
+<li>Set <code>prescribedHeatFlowRate=true</code> if there is a model connected to <code>heatPort</code>
```

Comment:
Both of the proposed settings are "true". Should one of them be "false" instead?